### ISSUE TITLE
T 72 fire gating not working properly

### ISSUE THREAD

**Issue by nord-kind** [RadianSimulations/ghpc-feedback#12]
**Describe the bug**
The T72 is able to fire its gun without alignment of main gun and sight.

**To Reproduce**
1. Play "T-72 Sniper" mission
2. Try to fire at the M60s in the distance as fast as possible after a stop
3. Notice how, even though the sight is well aligned, some shots will fly high or low.
4. Notice that this even applies to the Range finder, which will often return a false distance even though it is on target.

**Expected behavior**
T-72 should have fire gating, preventing the gun from firing. Also the Range finder should have no issue either.

**Desktop (please complete the following information):**
 - OS: Windows 7

**Additional context**
There was a comment some month ago that the fire gating error is due to the game not beeing able to update fast enough. Maybe it is possible to do some kind of work around in this case? Having the gun fire without any swinging, in a predetermined way is what all less sophisticated games do anyway, or not? (apart from gun offset of course)

**Comment by Lv100Latios** [comment]
Note: This sort of behavior seems to happens with any tank while stopping. While the tank is slowing down and is dipping forward, a tank with its gun looking right at a target will have the round fall short of the target. The opposite happens when starting to move with a tank, and appears to also be an issue while moving over rough terrain.

**Comment by nord-kind** [comment]
I confirm Lv100Latios notion. 
It is easiest to notive when using T-72 tanks though. Aim at a target far away. Lase it. Drive forward and constantly lase it (due to a bug constantly lazing still works in T-72). You willl notice the distance wheel jump back and forth several hundred meters, even though due to delta d, the distance should not change at all.

**Comment by JoshBusuito** [comment]
This is due to issues with the physical gun items stabilizing fast enough (frame perfect) under high rotation speed.  It is not a high priority and may or may not get fixed by accident if we rework some aiming code later on.

**Comment by nord-kind** [comment]
Here are 2 examples that show how this bug makes the FCS very unreliable.

Example - M1: https://youtu.be/qNE9yOLx0F0
Example - T-72: (in line to upload)

First part of each video: Notice how the lased range decreases and increases by several hundred meters when the suspension is working, even though the reticle is aimed at a constant target.

Second part of each video: Notice how this does affect the fired shells as well (notice the range for each shot is set right) . Notice that this behaviour is reliably repeatable.

In conclusion: short halt and shoot is completely not useable in the current state of the game. And therefore a basic tactic in tank combat is not useable.
